src/devices/machine/mb89372.cpp

```diff
@@ -0,0 +1,566 @@
+// license:BSD-3-Clause
+// copyright-holders:Ariane Fugmann
+/**
+    MB89372
+    Fujitsu
+    Multi-Protocol Controller
+ **/
+
+/*
+    based on guesswork!
+    port 00-0f - SIU A
+    port 10-1f - SIU B
+    port 20-2f - DMA A-D
+        20,21,22 chan a address (siu-a rx)
+        23       chan a command/status
+        24,25,26 chan b address (siu-a tx)
+        27       chan b command/status
+        28,29,2a chan c address (siu-b rx)
+        2b       chan c command/status
+        2c,2d,2e chan d address (siu-b tx)
+        2f       chan d command/status
+    port 30-3f - INT?
+*/
+
+#include "emu.h"
+#include "mb89372.h"
+#include "emuopts.h"
+
+//**************************************************************************
+//  LIVE DEVICE
+//**************************************************************************
+
+// device type definition
+DEFINE_DEVICE_TYPE(MB89372, mb89372_device, "mb89372", "MB89372 Multi-Protocol Controller")
```

Comment:
Long name needs `Fujitsu` prefixed.

Reply:
Alright, updated MB89372 and MB89374 accordingly.